package/network/services/openvpn-easy-rsa/Makefile

```diff
@@ -9,14 +9,22 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openvpn-easy-rsa
 
-PKG_VERSION:=3.0.4
+PKG_VERSION:=3.0.7
 PKG_RELEASE:=1
-PKG_SOURCE_URL:=https://github.com/OpenVPN/easy-rsa/releases/download/v$(PKG_VERSION)/
-PKG_SOURCE:=EasyRSA-$(PKG_VERSION).tgz
-PKG_HASH:=472167f976c6cb7c860cec6150a5616e163ae20365c81f179811d6ee0779ec5a
+PKG_SOURCE_URL:=https://github.com/OpenVPN/easy-rsa/archive/
```

Comment:
No. This filename is completely useless and potentially conflicts. Use codeload.

See the packages feed as an example.

Reply:
Done!